### PR TITLE
Fix issues in javadocs where > and < symbols have been used, resulting in javadoc errors.

### PR DESCRIPTION
**Pull request by JonathanGiles**
A number of JavaDoc issues were recently introduced, resulting in build failures for JavaDocs when running `mvn javadoc:aggregate`. This pull request fixes these.

### CODE REVIEW

File: azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/ICheckpointManager.java [hunk 1]
```diff
@@ -27,21 +27,21 @@
 	 * The returned CompletableFuture completes with true if the checkpoint store exists or false if it
 	 * does not. It completes exceptionally on error.
 	 * 
-	 * @return CompletableFuture -> true if it exists, false if not
+	 * @return CompletableFuture -&gt; true if it exists, false if not
```
Comment (sabeegrewal) [inline azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/ICheckpointManager.java]: I've used the @literal tag in the codebase. We should decide if we want to use @literal or HTML character codes. I don't have a preference either way, we should just be consistent.

Comment (JonathanGiles) [reply azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/ICheckpointManager.java]: I don't have a preference either. I've tended to use character codes in my past, but I don't prefer one over the other. Feel free to decline this PR if you'd rather use literal tags instead.

Comment (sabeegrewal) [reply azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/ICheckpointManager.java]: @sjkwak @SreeramGarlapati @JamesBirdsall

Comment (JonathanGiles) [reply azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/ICheckpointManager.java]: The main thing I want to see is that as part of the commit process we start to standardise on ensuring that `mvn javadoc:javadoc` completes without error. This way we can ensure that the repo is always able to generate API docs (which I am trying to do on a daily basis here: https://github.com/JonathanGiles/azure-javadocs)

Comment (sabeegrewal) [reply azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/ICheckpointManager.java]: Absolutely agree. The plan is to have Travis ensure a `mvn clean package` builds for javadocs and code before a PR can be merged.

**Comment by sabeegrewal** [comment]
Hey @JonathanGiles, we talked about it, and we're going to go with the literal tag since it's more legible. I'll get a PR in that fixes this issue.

**Comment by JonathanGiles** [comment]
Thanks - I see the PR is waiting to be merged. Once that is done I'll update my build to re-include event hubs.

**Comment by sabeegrewal** [comment]
@JonathanGiles the PR is merged! I updated Appveyor to make sure the javadocs don't have any errors as well. 👍

**Comment by JonathanGiles** [comment]
Thanks. I saw that and updated https://github.com/JonathanGiles/azure-javadocs - I have a build running now that should hopefully include event hubs again.

**Pull request closed without merging**